### PR TITLE
please review

### PR DESCRIPTION
**Pull request by ibobechko**

### CODE REVIEW

File: src/Iryna/OurFuture.java [hunk 1]
```diff
@@ -7,4 +7,8 @@ public class OurFuture {
     //Let's work on our future
 
     //so we can make some $$$$$
+
+    //do you want to learn?
```
Comment (alinazar77) [inline src/Iryna/OurFuture.java:11]: not really

File: src/Iryna/OurFuture.java [hunk 2]
```diff
@@ -7,4 +7,8 @@ public class OurFuture {
     //Let's work on our future
 
     //so we can make some $$$$$
```
Comment (alinazar77) [inline src/Iryna/OurFuture.java:9]: thats what i want

**Review by alinazar77 — COMMENTED** [review]
are everything good?

**Comment by ibobechko** [comment]
yes, you can approve if you like it

**Review by alinazar77 — COMMENTED** [review]
goof

**Review by alinazar77 — APPROVED** [review]
good

**Pull request merged**